aiohttp/http_writer.py

```diff
@@ -130,6 +130,7 @@ class PayloadWriter(AbstractPayloadWriter):
     def __init__(self, stream, loop, acquire=True):
         self._stream = stream
         self._transport = None
+        self._buffer = []
```

Comment:
if i remember correctly, aiohttp switched from bytearrays to list for performance reasons too, so this probably need to be benchmarked…